src/github.ts

```diff
@@ -0,0 +1,65 @@
+import { context, GitHub } from "@actions/github";
+import * as core from "@actions/core";
+import { Octokit } from "@octokit/rest";
+
+let octokitSingleton;
+
+export function getOctokitSingleton() {
```

Comment:
why not instantiating it directly?

Reply:
Because it will be instantiated as soon as I import `github.ts`. In tests this is annoying since I always need to mock it, even though I am not testing it.